chandra_aca/plot.py

```diff
@@ -191,45 +201,51 @@ def plot_stars(attitude, catalog=None, stars=None, title=None, starcat_time=None
         stars = agasc.get_agasc_cone(quat.ra, quat.dec,
                                      radius=1.5,
                                      date=starcat_time)
+
     if bad_stars is None:
         bad_stars = bad_acq_stars(stars)
 
     fig = plt.figure(figsize=(5.325, 5.325))
-    ax = fig.add_subplot(1, 1, 1)
-    plt.subplots_adjust(top=0.95)
+
+    # Start with an empty plot that just has the tick labels for yag/zag
+    ax_yz = fig.add_subplot(1, 1, 1)
```

Comment:
Because I like everything to be perfect, I have a little problem with this yag/zag grid because it is a lie.  But, I see it as a reasonable compromise to the history of using yag/zag.  Also from the code I see that while you are effectively just putting yag/zag labels on the axis, you used the old subplot to avoid doing it manually (getting the previous grid for free).